### ISSUE TITLE
Table CSS

### ISSUE THREAD

**Issue by yuki-kimoto** [metacpan/metacpan-web#2623]
Hi, thank you for the before design improvement.

This time, I want to add design to the table in the content.

```
<table>
  <tr><th>Header1</th><th>Header2</th></tr>
  <tr><td>Item11</td><td>Item12</td></tr>
  <tr><td>Item21</td><td>Item22</td></tr>
</table>
```

Border and the background color of headers.

Do you allow me to implement this?

**Comment by oalders** [comment]
Hi @yuki-kimoto. Do you have a screenshot of your proposed changes?

**Comment by yuki-kimoto** [comment]
Yes, I can create a screenshot of my proposed changes.

**Comment by yuki-kimoto** [comment]
I plan to write proposed change in a few weeks.

In this time, I try to serve meta cpan web in my environment.

**Comment by yuki-kimoto** [comment]
I succeed  to serve the metacpan-web today.

**Comment by yuki-kimoto** [comment]
I show a  a screenshot.

![podtablecss](https://user-images.githubusercontent.com/81858/145528165-ad832c1d-1a54-4f46-867e-c8a90da65dc2.png)

```
    /* root/static/less/pod.less */
    table {
        border-collapse:collapse;
    }
    
    tr {
      
    }
    
    th {
        text-align:center;
        background:#fafafa;
        font-weight:bold;
        padding: 5px 0;
        border:1px solid #ddd;
    }
    
    td {
        padding:5px 10px;
        border:1px solid #ddd;
    }
```

What do you think?

**Comment by oalders** [comment]
Thanks @yuki-kimoto. This looks fine to me, but I will defer to @haarg on this.

**Comment by yuki-kimoto** [comment]
OK. I wait for the answer of @haarg.

**Comment by haarg** [comment]
This should be fine - I've looked at various existing tables in pod on CPAN, and most I saw would benefit from this.

**Comment by yuki-kimoto** [comment]
@haarg 

Thank you for your reply.

Could I create the pull request?

**Comment by haarg** [comment]
If you could, yes please.

**Comment by yuki-kimoto** [comment]
I will create the pull request next week.

**Comment by yuki-kimoto** [comment]
I added the pull request.

#2638

**Comment by oalders** [comment]
Closed via #2638. Thanks, @yuki-kimoto!

**Comment by yuki-kimoto** [comment]
You are welcome!
